### PR TITLE
Put contours in groups, other minor fixes

### PR DESCRIPTION
**Pull request by rchristie**

### CODE REVIEW

File: tests/test_mbfxml2ex.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def test_write_tree_with_markers(self):
             os.remove(ex_file)
 
         write_ex(ex_file, neurolucida_data)
-        self.assertTrue(os.path.exists(ex_file))
```
Comment (hsorby) [inline tests/test_mbfxml2ex.py:224]: Probably want to keep this line as the next assert will assume it exists.

File: src/mbfxml2ex.py [hunk 2]
```diff
@@ -825,20 +837,21 @@ def load(region, data, options):
...
-            create_group(field_module, anatomical_name, element_ids)
+            create_group_elements(field_module, anatomical_name, element_ids)
         if element_ids and is_option('external_annotation', options):
             if options['external_annotation'] and 'anatomical term' in tree:
                 print('create external annotation for tree', tree['anatomical term'])
     for contour in data.get_contours():
         connectivity = determine_contour_connectivity(contour['data'], contour['closed'])
         node_identifiers = create_nodes(field_module, contour['data'])
-        field_info = {'rgb': contour['rgb'], 'annotation': contour['name']}
+        field_info = {'rgb': contour['rgb']}  # , 'annotation': contour['name']}
```
Comment (hsorby) [inline src/mbfxml2ex.py]: Remove the commented out code if it is going.

**Review by hsorby — CHANGES_REQUESTED** [review]

File: src/mbfxml2ex.py (hunk 2 above)
Comment (rchristie) [reply src/mbfxml2ex.py]: Done

File: tests/test_mbfxml2ex.py (hunk 1 above)
Comment (rchristie) [reply tests/test_mbfxml2ex.py:224]: Restored.

**Review by hsorby — APPROVED** [review]

**Pull request merged**